apis/v1alpha1/types.go

```diff
@@ -1,18 +1,15 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
 package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// A Conditioned may have conditions set or retrieved. Conditions typically
-// indicate the status of both a resource and its reconciliation process.
-// +kubebuilder:object:generate=false
-type Conditioned interface {
-	SetConditions(...metav1.Condition)
-	GetCondition(string) *metav1.Condition
-}
-
```

Comment:
also need to remove the interfaces in apis/placement/v1beta1/types.go

Reply:
I can't seem to find that file.